engines/wintermute/detection_tables.h

```diff
@@ -68,8 +68,8 @@ static const PlainGameDescriptor wintermuteGames[] = {
 	{"pigeons",         "Pigeons in the Park"},
 	{"projectdoom",     "Project: Doom"},
 	{"projectjoe",      "Project Joe"},
-	{"reversion1",      "Reversion: The Escape"},
-	{"reversion2",      "Reversion: The Meeting"},
+	{"reversion1",      "Reversion: Chapter 1 - The Escape"},
```

Comment:
The in-game window-title just says "Reversion - Wintermute Engine". The in-game title screen says "Reversion: Chapter 1 - The Escape.